physical_validation/tests/test_util_kinetic_energy.py

```diff
@@ -0,0 +1,119 @@
+###########################################################################
+#                                                                         #
+#    physical_validation,                                                 #
+#    a python package to test the physical validity of MD results         #
+#                                                                         #
+#    Written by Michael R. Shirts <michael.shirts@colorado.edu>           #
+#               Pascal T. Merz <pascal.merz@colorado.edu>                 #
+#                                                                         #
+#    Copyright (C) 2012 University of Virginia                            #
+#              (C) 2017 University of Colorado Boulder                    #
+#                                                                         #
+#    This library is free software; you can redistribute it and/or        #
+#    modify it under the terms of the GNU Lesser General Public           #
+#    License as published by the Free Software Foundation; either         #
+#    version 2.1 of the License, or (at your option) any later version.   #
+#                                                                         #
+#    This library is distributed in the hope that it will be useful,      #
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of       #
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU    #
+#    Lesser General Public License for more details.                      #
+#                                                                         #
+#    You should have received a copy of the GNU Lesser General Public     #
+#    License along with this library; if not, write to the                #
+#    Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,     #
+#    Boston, MA 02110-1301 USA                                            #
+#                                                                         #
+###########################################################################
+r"""
+This file contains tests for the `physical_validation.util.kinetic_energy` module.
+"""
+import itertools
+
+import pytest
```

Comment:
If we use pytest only in tests - should we still add it to requirements.txt?

Reply:
I'm not sure there is a canonized answer to this, but my opinion is basically no; I prefer that the contents of `requirements.txt` contain what is needed for a user to access the core API of a package and test-only dependencies are outside that scope.